client/my-sites/domains/components/mapping-instructions/index.jsx

```diff
@@ -30,13 +30,113 @@ class DomainMappingInstructions extends React.Component {
 		areDomainDetailsLoaded: PropTypes.bool,
 		domainName: PropTypes.string,
 		isAtomic: PropTypes.bool,
+		subdomainPart: PropTypes.string,
+		wpcomDomainName: PropTypes.string,
 	};
 
 	static defaultProps = {
 		areDomainDetailsLoaded: false,
 		isAtomic: false,
 	};
 
+	renderNameServerInstructions() {
+		const { translate } = this.props;
+
+		const nameServerInstructionsMessage = translate(
+			'Please log into your domain name registrar account and {{strong}}update the name servers{{/strong}} of your domain to use the following values, as detailed in {{link}}these instructions{{/link}}:',
```

Comment:
```suggestion
			'Please log into your domain name registrar account and {{strong}}add NS records{{/strong}} for you subdomain to use following values, as detailed in {{link}}these instructions{{/link}}:',
```

And then show the proper DNS record format - I believe the one @klimeryk provided is correct though the subdomain there should be FQDN ending with dot. So:
```
blog.example.com. IN NS ns1.wordpress.com
blog.example.com. IN NS ns2.wordpress.com
blog.example.com. IN NS ns3.wordpress.com
```

Reply:
And yeah - we should definitely ask the guild to update the instructions for mapping a subdomain ( https://wordpress.com/support/domains/map-subdomain/ ) to include the NS records setup instructions and link to it